### PR TITLE
Remove lint

### PR DESCRIPTION
**Pull request by kortschak**
Mainly names.

### CODE REVIEW

**Comment by kortschak** [comment]
PTAL

File: search/internals.go [hunk 1]
```diff
@@ -94,17 +94,17 @@ func setupFuncs(g graph.Graph, cost graph.CostFunc, heuristicCost graph.Heuristi
 
 /** Sorts a list of edges by weight, agnostic to repeated edges as well as direction **/
 
-type edgeSorter []concrete.WeightedEdge
+type byWeight []concrete.WeightedEdge
 
-func (e edgeSorter) Len() int {
+func (e byWeight) Len() int {
```
Comment (sbinet) [inline search/internals.go]: minor nit: should the receiver value by named something else than `e`?
(`sort.{Int,Float,String}Slice` receivers are named `p`.)

**Comment by sbinet** [comment]
apart from the above minor nit: LGTM.

File: search/internals.go (hunk 1 above)
Comment (kortschak) [inline search/internals.go]: I don't think there is a strong convention on this; have a look at the examples in sort. The two choices I would make are from {'e', 'w'}. I don't think it matters that much.

Comment (sbinet) [inline search/internals.go]: agreed.

**Comment by Jragonmiris** [comment]
:+1:

**Pull request merged**